doc/man/salt-ssh.1

```diff
@@ -177,6 +177,11 @@ Specify the SSH private key file to be used for authentication.
 .UNINDENT
 .INDENT 0.0
 .TP
+.B \-\-priv\-passwd=SSH_PRIV_PASSWD
+Specify the SSH private key file's passphrase.
```

Comment:
Can you remove this change?

We compile these when they are packaged, it does not need to be updated.